docs/source/usage/plugins.rst

```diff
@@ -61,6 +61,16 @@ Examples
 * ``42,30:50:10``: at steps 30 40 42 50 84 126 168 ...
 * ``5,10``: at steps 0 5 10 15 20 25 ... (only executed once per step in overlapping intervals)
 
+Python Preprocessing
```

Comment:
ah, sorry but this does not belong to plugins :)
Maybe add this intro to `usage-python-utils` where you have the docs for the python utils already?
Maybe the section of workflows is sufficient as well?

Reply:
yeah, I was a little confused where to put it but I guess it works if I just show the example in the workflows and then link to usage-python-utils as you suggest